src/pages/demo/cody.tsx

```diff
@@ -15,7 +15,6 @@ import {
 } from '../../components'
 import { useAuthModal } from '../../context/AuthModalContext'
 import { breakpoints } from '../../data/breakpoints'
-import { EventName, getEventLogger } from '../../hooks/eventLogger'
 import { useWindowWidth } from '../../hooks/windowWidth'
```

Comment:
Also lets fix the error on `line 1` that is causing the yarn lint fail:  "'useEffect' is defined but never used"